package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "react-native-modal-filter-picker",
-  "version": "1.3.2",
+  "version": "1.3.3",
```

Comment:
Please undo this version change. This is likely going to have to be a major release.

Reply:
done

Reply:
```suggestion
  "version": "1.3.3",
```
done